### ISSUE TITLE
Filtering: Subscriber comment not caught

### ISSUE THREAD

**Issue by Firecul** [ThioJoe/YT-Spammer-Purge#690]
### Filter Mode

Auto-Smart Mode

### Select the Problem

A type of spammer is not detected at all

### (Optional) If 'Other', Enter Very Short Description

One of the begging for subscribers comments slips through, caught by duplicate detection.

### Spammer Example / Sample

![WindowsTerminal_Windows_PowerShell (37)](https://user-images.githubusercontent.com/34950371/155037161-cc4853e3-b71d-4c9e-b68c-1f32adec8210.png)

### Video / Post Link

https://www.youtube.com/watch?v=bVk8XyjhTKo&lc=UgxKT1fW-IkQLLkDU8N4AaABAg.9XxnN7J_2fJ9Xxnl6MEjXZ

**Comment by ThioJoe** [comment]
I'm afraid it is probably impossible to catch every single one of these because they rephrase it too much, so the duplicate checker will have to be sufficient
